src/chi/components/card/card.scss

```diff
@@ -124,7 +124,30 @@
       &.-noBorder {
         box-shadow: none;
       }
+    }
+
+    &.-empty {
+      background-color: transparent;
+      border: dashed rgba(set-color(black), 0.12);
+      transition: all 0.2s;
+
+      & .a-card__content {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin: 1rem;
+
+        & .a-icon {
+          color: set-color(grey, 60);
+          margin-bottom: 1rem;
+        }
 
+        & span {
```

Comment:
This would apply to every span tag. We need something more specific. I'd recommend creating a class or removing and using a flat button instead. @jllr Thoughts?

Reply:
@mattnickles @davidojedacl makes sense. I would add a class named .label or .title for the empty states.